web/src/features/auth/components/AuthWrapper.tsx

```diff
@@ -25,11 +25,13 @@ const setAxiosRequestInterceptors = (): AppThunk => (_, getState) => {
 const AuthWrapper = ({ children }: Props) => {
   const dispatch: AppDispatch = useDispatch()
   const { isAuthenticated, authenticating, error } = useSelector(selectAuthentication)
+  const didInit = useRef(false)
 
   useEffect(() => {
     if (TEST_AUTH || window.Cypress) {
       dispatch(testAuthenticate(true, 'test token', 'test id token'))
-    } else {
+    } else if (!didInit.current) {
```

Comment:
We may want to consider a singleton pattern less dependent on react useRef, something like: https://refactoring.guru/design-patterns/singleton/typescript/example

Something like we did with the `MoreCast2Column`